llm/context/nostr-protocol.md

```
@@ -1,269 +1,329 @@
-Every Nostr user has a keypair:
-- **Private Key (nsec):** Secret key for signing, never shared
-- **Public Key (npub):** Identity, can be shared
+Gift wrap is the foundation of BitcoinLink's security model. It provides multi-layer encryption with metadata hiding.
 
-```javascript
-import { generatePrivateKey, getPublicKey } from 'nostr-tools';
+### Structure
 
-const sk = generatePrivateKey();  // Private key (Uint8Array)
-const pk = getPublicKey(sk);      // Public key (hex string)
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)